MyCryptoHQ/MyCrypto
# Fixes some issues with wallets for RSK

Closes #2213

### Description

There is a bug in `common/features/config/selectors.ts` that doesn't allow to have available some wallets for RSK.
There is also a bug in `RSK_TESTNET` configuration that enables Safe-T wallet and it shouldn't be.

### Changes

* Condition in `common/features/config/selectors.ts` 
* Configuration in `RSK_TESTNET`

### Steps to Test

1. Choose `RSK` network (or `RSK_TESTNET` network)
2. Go to Account tab
3. Check that every wallet is available but Parity and Safe-T
